File: level_2/피보나치_수/devSOWON0628.js
function solution(n) {
    // 피보나치 수열의 첫번째(index:0), 두번째(index:1)의 값은 1이다.
    let f = [1, 1]
    for(let i = 2 ; i < n ; i++){
        f[i] = (f[i-1] + f[i-2]) % 1234567
    }
    return f[f.length-1]
}

File: level_2/n^2_배열_자르기/devSOWON0628.java
import java.util.*;

class Solution {
    public int[] solution(int n, long left, long right) {
        int size = (int) (right - left + 1); // 결과 배열 크기
        int[] result = new int[size];

        for (long index = left; index <= right; index++) {
            int i = (int) (index / n);  // 행 번호
            int j = (int) (index % n);  // 열 번호
            result[(int) (index - left)] = Math.max(i + 1, j + 1);
        }

        return result;
    }
}
